src/components/DelegateMultipleButton.jsx

```diff
@@ -339,17 +328,157 @@ const DelegateMultipleButton = props => {
+                  helpText=""
+                  value={selectedToken && selectedToken.address}
+                  options={tokenWhitelistOptions}
+                  onChange={address => setToken(address)}
+                />
+              )}
+
+              {delegations.length === 0 || maxAmount.isZero() ? (
+                <p>
+                  The amount available to delegate is 0 {selectedToken.symbol}
+                  <br />
+                  Please select{' '}
+                  {!props.milestone || !props.milestone.acceptsSingleToken
+                    ? 'a different currency or '
+                    : ''}
+                  different source {milestone ? 'DAC/Campaign' : 'DAC'}
+                </p>
+              ) : (
+                <div>
+                  <span className="label">Amount {selectedToken.symbol} to delegate:</span>
+
+                  <div className="form-group" id="amount_slider">
+                    <Slider
+                      min={0}
+                      max={maxAmount.toNumber()}
+                      onChange={num => setAmount(num.toString())}
+                      value={amount}
+                      step={decimals ? 1 / 10 ** decimals : 1}
+                      marks={sliderMarks}
+                    />
+                  </div>
+
+                  <div className="form-group">
+                    <NumericInput
+                      onChange={setAmount}
+                      token={selectedToken}
+                      value={amount}
+                      lteMessage={`The donations you are delegating have combined value of ${maxAmount.toNumber()}. Do not input higher amount than that.`}
+                      id="amount-input"
+                      maxAmount={maxAmount}
+                    />
+                  </div>
+                  <div className="form-group">
+                    <Textarea name="comment" id="comment-input" value="" placeholder="Comment" />
+                  </div>
+                  <button
+                    className="btn btn-success"
+                    formNoValidate
+                    type="submit"
+                    disabled={isSaving || !isForeignNetwork || !formIsValid}
+                  >
+                    {isSaving ? 'Delegating...' : 'Delegate here'}
+                  </button>
+                  <button
+                    className="btn btn-light float-right"
+                    type="button"
+                    onClick={() => {
+                      setModalVisible(false);
+                    }}
+                  >
+                    Close
+                  </button>
+                </div>
+              )}
+            </div>
+          )}
+        </Form>
+      </Fragment>
+    </Fragment>
+  );
+
+  const modalLoading = (
+    <div>
+      <h2>Please wait while data is loading</h2>
+      <Loader />
+    </div>
+  );
+
+  const isContextReady = !whiteListIsLoading && !userContextIsLoading && Web3ContextIsEnabled;
```

Comment:
I wish isContextReady not be true till the getDacs fetched and filled the options

Reply:
done